File: api/errors/templates/template_not_found_error.py
from api.enums.system import ErrorCodeEnum
from api.errors.defaults import BaseError


class TemplateNotFoundError(BaseError):
    def __init__(self, message: str | None = None) -> None:
        super().__init__(ErrorCodeEnum.TEMPLATE_NOT_FOUND_ERROR, message or "Template not found")

File: web/errors/builds/TemplateNotFoundError.ts
// enums
import { ErrorCodeEnum } from '@/enums/common';

// errors
import { BaseError } from '@/errors/_base';

export default class TemplateNotFoundError extends BaseError {
  public readonly code: ErrorCodeEnum = ErrorCodeEnum.TemplateNotFoundError;
  public readonly displayName: string = 'TemplateNotFoundError';
}
